vendor/github.com/elastic/gosigar/sigar_windows.go

```diff
@@ -348,6 +348,10 @@ func (self *ProcTime) Get(pid int) error {
 	return nil
 }
 
+func (self *ProcIO) Get(pid int) error {
+	return ErrNotImplemented{runtime.GOOS}
```

Comment:
Thanks for your reply.
The sigar_freedsb.go mention that "Copied and modified from sigar_linux.go", so I should change which one ?